### PR TITLE
added ACR dummy detector template

### PR DESCRIPTION
**Pull request by mksuni**

### CODE REVIEW

File: data/templates/Detector_ContainerRegistry.csx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using System.Threading;
+
+private static string GetQuery(OperationContext<ArmResource> cxt)
+{
+    return
+    $@"
+		let startTime = datetime({cxt.StartTime});
+		let endTime = datetime({cxt.EndTime});
+		cluster('ClusterName').database('DBName').YOUR_TABLE_NAME
+		| where Timestamp == startTime and Timestamp == endTime
```
Comment (nmallick1) [inline data/templates/Detector_ContainerRegistry.csx]: change this line to following ?
| where TIMESTAMP >= startTime and TIMESTAMP <= endTime

File: data/templates/Detector_ContainerRegistry.csx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+
+[ArmResourceFilter(provider: "Microsoft.ContainerRegistry", resourceTypeName: "registries")]
+[Definition(Id = "YOUR_DETECTOR_ID", Name = "", Author = "YOUR_ALIAS", Description = "")]
+public async static Task<Response> Run(DataProviders dp, OperationContext<ArmResource> cxt, Response res)
+{
+    res.Dataset.Add(new DiagnosticData()
+    {
+        Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt)),
```
Comment (nmallick1) [inline data/templates/Detector_ContainerRegistry.csx]: Change this line to 
Table = await dp.Kusto.ExecuteClusterQuery(GetQuery(cxt), null, "GetQuery"),

Comment (mksuni) [reply data/templates/Detector_ContainerRegistry.csx]: done

File: data/templates/Detector_ContainerRegistry.csx (hunk 1 above)
Comment (mksuni) [reply data/templates/Detector_ContainerRegistry.csx]: done

**Pull request merged**